instrumentation/spring/spring-boot-resources/library/src/test/resources/build-info.properties

```diff
@@ -1,2 +1 @@
 build.artifact=something
-build.name=some-name
```

Comment:
Cold you please explain the removal of this line?

Reply:
this is a test file that is supposed to not find the expected entry.
Before, it did not have `build.version` - and now it also must not have `build.name`.